From 6ed3b2245a0560938dbce51e91e128e0a8ced965 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 22 Nov 2013 16:37:10 +0000
Subject: [PATCH 130/462] Change, Mantis 0029874: LogManager: use new
 PlatformStatus.idl

Build.xml
V1.2.0

AntProperties
use new IDL directory in SelfServicePlatform for IDL compiler


git-svn-id: svn://localhost/SelfServiceBuild/trunk@192 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../Build/java/AntBuild.ReleaseNotes.json     | 21 +++++++++++++++++++
 SelfServiceBuild/Build/java/AntProperties.xml |  2 +-
 2 files changed, 22 insertions(+), 1 deletion(-)
 create mode 100644 SelfServiceBuild/Build/java/AntBuild.ReleaseNotes.json

diff --git a/SelfServiceBuild/Build/java/AntBuild.ReleaseNotes.json b/SelfServiceBuild/Build/java/AntBuild.ReleaseNotes.json
new file mode 100644
index 0000000..9d6d3a0
--- /dev/null
+++ b/SelfServiceBuild/Build/java/AntBuild.ReleaseNotes.json
@@ -0,0 +1,21 @@
+{
+ "Module": ["build.xml"],
+ "ChangeLog":[
+  {
+   "Version": "1.2.0",
+   "Date": "2013-11-22",
+   "Author": "SteT",
+   "Ticket": [
+    {
+     "System": "MantisIntern",
+     "Ticket": 29874,
+     "Type": "Change",
+     "Abstract": "LogManager: use new PlatformStatus.idl"
+    }
+   ],
+   "Change": [
+    {"Comment": "AntProperties: use new IDL directory in SelfServicePlatform for IDL compiler"}
+   ]
+  }
+ ]
+}
diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 4167e82..368187c 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -69,7 +69,7 @@
 <!-- IDL-Verzeichnisse -->
 <property name="CommonMassaiIdlDir"         value="${COMMONROOT}/Massai/idl/Massai"/>
 <property name="PlatformCussIdlDir"         value="${CUSSROOT}/Massai/idl/Cuss"/>
-<property name="PlatformMassaiIdlDir"       value="${CUSSROOT}/Massai/idl/Massai"/>
+<property name="PlatformMassaiIdlDir"       value="${PLATFORMROOT}/Massai/idl/Massai"/>
 <property name="AblsMassaiIdlDir"           value="${ABLSROOT}/Massai/idl/Massai"/>
 <property name="AppMassaiIdlDir"            value="${APPROOT}/Massai/idl/Massai"/>
 
-- 
2.41.0.windows.1

